docs/source/config.json

```diff
@@ -39,14 +38,14 @@
       "Error handling": "/essentials/errors",
       "Custom scalars": "/essentials/custom-scalars",
       "Fragments": "/essentials/fragments",
-      "@defer support (experimental)": "/fetching/defer",
+      "@defer support": "/fetching/defer",
```

Comment:
I think we still want `@defer` to be experimental on mobile until the spec is merged

Reply:
Oh, I see below you removed the (experimental) from all left panel items. Is that on purpose?

Reply:
👍 web and ios don't have (experimental) in the menu. LGTM ✅